impeller/entity/contents/linear_gradient_contents.h

```diff
@@ -53,6 +53,10 @@ class LinearGradientContents final : public ColorSourceContents {
                   const Entity& entity,
```

Comment:
Can we also remove RenderTexture in this patch?

Reply:
Eventually, I want to leave the old stuff around while I get the new stuff working and then I can run comparisons for testing/benchmarking.

Reply:
SGTM

Reply:
Per other discussions I think we decided it would stay in case we get more than 256 colors. Eventually we hope to have a vertex tiling solution that works efficiently on all platforms, but until then we'll still have the issue with zero-length stops on older platforms, but only if there are more than 256 stops total...